src/css/profile/mobile/fonts.less

```diff
@@ -1,16 +1,18 @@
 @font-face {
-    font-family: Roboto-Light;
+    font-family: sans-serif-light;
     src: url(fonts/Roboto-Light.ttf);
 }
 
 @font-face {
-    font-family: Roboto-Regular;
+    font-family: sec-roboto-light;
```

Comment:
according to the MDN font-family
Specifies a name that will be used as the font face value for font properties.
if we have two the same font-family names how we can differentiate them?